Packs/Armis/Integrations/Armis/Armis.yml

```
@@ -207,37 +201,37 @@ script:
     execution: false
     name: armis-search-alerts-by-aql-string
     outputs:
-    - contextPath: Armis.SearchAlertsByAQLString.activityIds
+    - contextPath: Armis.Armis.activityIds
```

```suggestion
    - contextPath: Armis.Alert.activityIds
```
I think you meant this path